gobblin-data-management/src/main/java/gobblin/data/management/copy/CopySource.java

```diff
@@ -122,82 +122,22 @@
     this.metricContext = Instrumented.getMetricContext(state, CopySource.class);
 
     try {
-
-      final FileSystem sourceFs = getSourceFileSystem(state);
```

Comment:
I see a lot of code moved around but I don't see any change in logic in CopySource. Is that correct? If not, can you please describe the nature of the change?